ASP_tests.py

```diff
@@ -259,6 +259,76 @@ def test_plot_lc():
     assert output[5] == 0.0
 
 
+def test_regular_grid():
+    from AllASPFuncs import local_grid
+    wcs = np.load('./tests/testdata/wcs_dict.npz', allow_pickle=True)
+    wcs = dict(wcs)
+    ra_center = wcs['CRVAL1']
+    dec_center = wcs['CRVAL2']
+    for key in wcs.keys():
+        wcs[key] = wcs[key].item()
+    wcs = galsim.wcs.readFromFitsHeader(wcs)[0]
+    ra_grid, dec_grid = local_grid(ra_center, dec_center, wcs,
+                                   size=25, spacing=3.0, image=None,
+                                   percentiles=[],
+                                   makecontourGrid=False)
+    test_ra = np.array([7.67363133, 7.67373506, 7.67383878, 7.67355803,
```

Comment:
This might be a bit more readable if you picked a "central" RA value and then added in offsets around it with respect to ra_central and dec_central.
```
ra_central = 7.673
test_ra = ra_center + np.array([0.00063133, 0.00073506, ...])
```
But this is a pretty straightforward test case.  No need to change.  Just an idea to consider.